### ISSUE TITLE
Regard player context objects as immutable after setup

### ISSUE THREAD

**Issue by michael-buschbeck** [michael-buschbeck/mychs-macro-magic#98]
Player context objects must be immutable after initial setup so that existing references (attached to already-queued MMM commands) aren't changed when the context changes later.

This was changed in version 1.17.2 and should be reverted.

**Comment by michael-buschbeck** [comment]
To reproduce:
```
!mmm script
!mmm     set roll1 = [[1d1]]
!mmm     set roll2 = [[2d1]]
!mmm     chat: ${roll1}, ${roll2}
!mmm end script
```

Should output `1, 2`, does output `2, 2`. Duh.

**Comment by michael-buschbeck** [comment]
...and the reason it "worked" in our test game was that my MychTest.js sandbox script there had residual test code that just happened to "fix" the bug in MMM 1.17.2. Double-Duh.
